monitoring/main.libsonnet

```diff
@@ -0,0 +1,3 @@
+(import 'alerts/cockroachdb.libsonnet') +
```

Comment:
Could this file be named `mixin.libsonnet` instead of `main.libsonnet`? This way it would be consistent with other mixins.

Reply:
Yeah. I though about it, but feel like this is nicer. Would you need to change a lot to support it?